src/rsvc/srv.c

```diff
@@ -1169,11 +1173,12 @@ bcast_create(crt_opcode_t opc, bool filter_invert, d_rank_list_t *filter_ranks,
  * \param[in]	create		create replicas first
  * \param[in]	bootstrap	start with an initial list of replicas
  * \param[in]	size		size of each replica in bytes if \a create
+ * \param[in]	vos_df_version	version of VOS durable format if \a create
```

Comment:
No change requested in this patch. I don't think we have any callers that specify create=false.

Reply:
@kccain, good observation. Unfortunately, the CR landing has changed that: There's a non-create caller in `rdbt_dictate_handler`. Once the base branch lands, I will have to rebase this branch and fix the conflicts with the CR landing. We'll then have the non-create caller.